e2e-tests/cypress/tests/cypress/e2e/models/ForgotPasswordPage.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+
+export class ForgotPasswordPage {
+  // selectors
+  emailInput = "input[type=email]";
+  submitBtn = "button[type=submit]";
+  successComponent = "[data-test='forgot-password-success']";
```

Comment:
```suggestion
  emailInput = 'input[type=email]'
  submitBtn = 'button[type=submit]'
  successComponent = '[data-test="forgot-password-success"]'
```

Reply:
Sure, this will be handled in a seperate task to adapt the linting to our repo's standards.